internal/pkg/deploy/cloudformation/stack/lb_web_svc.go

```diff
@@ -304,15 +297,30 @@ func (s *LoadBalancedWebService) Parameters() ([]*cloudformation.Parameter, erro
 			ParameterKey:   aws.String(LBWebServiceTargetPortParamKey),
 			ParameterValue: targetPort,
 		},
-		{
-			ParameterKey:   aws.String(LBWebServiceStickinessParamKey),
-			ParameterValue: aws.String(strconv.FormatBool(aws.BoolValue(s.manifest.Stickiness))),
-		},
 		{
 			ParameterKey:   aws.String(WorkloadEnvFileARNParamKey),
 			ParameterValue: aws.String(s.rc.EnvFileARN),
 		},
-	}...), nil
+	}...)
+
+	if !s.manifest.RoutingRule.Disabled() {
+		httpParams := []*cloudformation.Parameter{
+			{
+				ParameterKey:   aws.String(LBWebServiceRulePathParamKey),
+				ParameterValue: s.manifest.RoutingRule.Path,
+			},
+			{
+				ParameterKey:   aws.String(LBWebServiceHTTPSParamKey),
+				ParameterValue: aws.String(strconv.FormatBool(s.httpsEnabled)),
+			},
+			{
+				ParameterKey:   aws.String(LBWebServiceStickinessParamKey),
+				ParameterValue: aws.String(strconv.FormatBool(aws.BoolValue(s.manifest.RoutingRule.Stickiness))),
+			},
+		}
+		wkldParams = append(wkldParams, httpParams...)
```

Comment:
It seems more straightforward to me to have
```suggestion
		wkldParams = append(wkldParams, []*cloudformation.Parameter{...}...)
```
🤔